Specs/NumberWithUnit/Dutch/TemperatureModel.json

```diff
@@ -416,7 +415,7 @@
         }
       },
       {
-        "Text": "graden celsius",
+        "Text": "celsius",
```

Comment:
Start/end are also missing in all spec cases?
Please open an issue to make sure the test code enforces they exist in the spec and match the engine output.
Could you also add all to the spec file?

Reply:
added Start/Lenght to spec